packages/server/next/app/env.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+"use client";
+
+/**
+ * A helper function to get environment variables from the window object
```

Comment:
Wrong doc. These are not environment variables. Then also getEnv is confusing. getConfig ?